jest.config.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
-  projects: [
```

Comment:
`globalSetup` and `globalTeardown` didn't work within projects, maybe this should be added as a issue with jest.